### DISCUSSION TITLE
i want to send money to show my appreciation of this tool

### DISCUSSION THREAD

**Discussion by michapixel** [tranxuanthang/lrcget#57] [General]
how (apart from buy me a coffeee) can i do this?

you're awesome!!! ❤️❤️❤️

**Comment by tranxuanthang** [comment]
Thank you for considering! Unfortunately there aren't many ways to receive donations across countries conveniently and cheaply.

Aside from Buy Me A Coffee, there are cryptocurrency options: Monero and Litecoin. But if you are not familiar with cryptocurrencies, you might need some time to learn how to send money using them. Another option is GitHub Sponsors, but the last time I checked, it did not support my country (I will do a recheck soon when I have some spare time).

Could you suggest any methods that are convenient for you in your location?

**Comment by michapixel** [comment]
what about paypal?

**Comment by tranxuanthang** [comment]
I'll give it a try! I've added Paypal link in README:

https://github.com/tranxuanthang/lrcget?tab=readme-ov-file#donation
